storage/storagepb/kv.proto

```diff
@@ -14,11 +14,14 @@ message KeyValue {
   int64 create_revision = 2;
   // mod_revision is the last modified revision of the key.
   int64 mod_revision = 3;
+  // tombstone_revision is the tombstone revision of the key.
+  // If the key is not tombstoned, its revision is set to zero.
+  int64 tombstone_revision = 4;
```

Comment:
We do not return a deleted key to user from KV interface.

It returns a deleted key when users watch on the key and the key is deleted, and users will see the tombstone_rev in this case.

It helps users to know when the deletion happens, and it works similarly as mod_revision.